--- ./node_modules/ionic-native/dist/esm/plugins/bluetoothserial.js.bak	2017-03-21 13:33:05.341532690 +0100
+++ ./node_modules/ionic-native/dist/esm/plugins/bluetoothserial.js	2017-03-22 09:44:08.984900656 +0100
@@ -47,6 +47,11 @@
      */
     BluetoothSerial.connectInsecure = function (macAddress) { return; };
     /**
+     * Disconnect from the connected device
+     * @returns {Promise<any>}
+     */
+    BluetoothSerial.disconnect = function () { return ;};
+    /**
      * Writes data to the serial port
      * @param {any} data ArrayBuffer of data
      * @returns {Promise<any>} returns a promise when data has been written
@@ -150,6 +155,11 @@
     ], BluetoothSerial, "connectInsecure", null);
     __decorate([
         Cordova({
+            platforms: ['Android', 'iOS', 'Windows Phone'],
+        })
+    ], BluetoothSerial, "disconnect", null);
+    __decorate([
+        Cordova({
             platforms: ['Android', 'iOS', 'Windows Phone']
         })
     ], BluetoothSerial, "write", null);
@@ -252,4 +262,4 @@
     ], BluetoothSerial);
     return BluetoothSerial;
 }());
-//# sourceMappingURL=bluetoothserial.js.map
\ No newline at end of file
+//# sourceMappingURL=bluetoothserial.js.map
