--- a/node_modules/ionic-native/dist/esm/plugins/bluetoothserial.d.ts	2017-03-21 13:30:05.442266222 +0100
+++ b/node_modules/ionic-native/dist/esm/plugins/bluetoothserial.d.ts	2017-03-21 13:35:25.752082152 +0100
@@ -39,6 +39,11 @@
      */
     static connectInsecure(macAddress: string): Observable<any>;
     /**
+     * Disconnect from the connected device
+     * @returns {Promise<any>}
+     */
+    static disconnect(): Promise<any>;
+    /**
      * Writes data to the serial port
      * @param {any} data ArrayBuffer of data
      * @returns {Promise<any>} returns a promise when data has been written
